src/test/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowServiceTest.java

```diff
@@ -337,6 +337,22 @@ public void createCase_whenHasOrganisationCorrespondentDetails() {
         caseworkClient.getCorrespondentsForCase(caseUUID);
     }
 
+    @Test
+    public void migrateCase() {
+        String caseDataType = "BF";
+        LocalDate dateReceived = LocalDate.EPOCH;
+        List<DocumentSummary> documents =  new ArrayList<>();
+        UUID userUUID = UUID.randomUUID();
+        Map<String, String> receivedData = new HashMap<>();
+        MigrateCaseworkCaseResponse migrateCaseworkCaseResponse = new MigrateCaseworkCaseResponse(UUID.randomUUID(), null);
+
+        when(caseworkClient.migrateCase(any(), any(), any(), any())).thenReturn(migrateCaseworkCaseResponse);
+
+        MigrateCaseResponse output = workflowService.migrateCase(caseDataType, dateReceived, documents, userUUID,null, receivedData);
+        assertThat(output.getUuid()).isNotNull();
+        verify(camundaClient, times(1)).startCase(any(), any(), any());
+        verify(caseworkClient, times(0)).saveCorrespondent(any(), any(), any());
```

Comment:
I don't think this test is really testing the logic of the method here... `migrateCase` does a whole bunch of data mapping logic that your test isn't catching. Neither is it verifying that the caseworkClient method to migrate is called.